kubernetes/client.go

```diff
@@ -29,7 +33,15 @@ type simpleClient struct {
 	httpClient *http.Client
 }
 
-const defaultControllerUserAgent = "kube-ingress-aws-controller"
+const (
+	defaultControllerUserAgent = "kube-ingress-aws-controller"
+
+	// aligned with https://pkg.go.dev/net/http#DefaultTransport
+	maxIdleConns        = 100
```

Comment:
maxIdleConns should be less, maybe 10. As we only deal with kubernetes api server here and we do not expect too many concurrent requests

Reply:
makes sense, thanks